crates/holochain/src/core/ribosome/host_fn/hash.rs

```diff
@@ -19,6 +20,9 @@ pub fn hash(
         HashInput::Header(header) => HashOutput::Header(
             holochain_zome_types::header::HeaderHashed::from_content_sync(header).into_hash(),
         ),
+        HashInput::External(bytes) => HashOutput::External({
+            ExternalHash::from_raw_32(holo_hash::encode::blake2b_256(&bytes))
```

Comment:
This is a bit weird but maybe it's ok for this to block because all zome calls are made from within a spawn_blocking anyway?